css/index.less

```diff
@@ -1,3 +1,4 @@
+@import (inline) '../node_modules/normalize-css/normalize.css';
```

Comment:
I was hoping to include normalize.css without copying it into the build directory. The `inline` keyword causes the contents of normalize to be merged into the final CSS.

I'm not sure if this is a good approach. Maybe better not to do that.